### DISCUSSION TITLE
Sponsor

### DISCUSSION THREAD

**Discussion by guiaramos** [nextauthjs/next-auth#655] [Ideas]
**Your question**
Is there anyway to sponsor the project? Ex. GitHub Sponsors

**What are you trying to do**
Sponsor the project

**Feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [ ] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [ ] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by stale** [comment]
Hi there! It looks like this issue hasn't had any activity for a while. It will be closed if no further activity occurs. If you think your issue is still relevant, feel free to comment on it to keep ot open. Thanks!

**Comment by balazsorban44** [comment]
Hi there! So good to here you would like to sponsor the project! Very appreciated!
I don't think we currently have anything set up. I am going to note it for discussion, and check back when I have some information.

**Comment by balazsorban44** [comment]
@guiaramos, we now actually have an OpenCollective page, if you are still interested: https://opencollective.com/nextauth :green_heart:
